.github/CODEOWNERS

```diff
@@ -30,11 +30,11 @@ sdk/src/azure/core/           @ahsonkhan @antkmsft @rickwinter @vhvb1989 @gearam
 sdk/tests/core/               @ahsonkhan @antkmsft @rickwinter @vhvb1989 @gearama
 
 # IoT
-sdk/docs/iot/                 @CIPop @danewalton @ericwol-msft @ewertons @jspaith @momuno @preethi826
-sdk/inc/azure/iot/            @CIPop @danewalton @ericwol-msft @ewertons @jspaith @momuno @preethi826
-sdk/samples/iot/              @CIPop @danewalton @ericwol-msft @ewertons @jspaith @momuno @preethi826
-sdk/src/azure/iot/            @CIPop @danewalton @ericwol-msft @ewertons @jspaith @momuno @preethi826
-sdk/tests/iot/                @CIPop @danewalton @ericwol-msft @ewertons @jspaith @momuno @preethi826
+sdk/docs/iot/                 @CIPop @danewalton @ericwol-msft @ewertons @jspaith @vaavva @preethi826
```

Comment:
nit: she should be the last name to keep the alphabetical

Reply:
Feel free to make the change :).  I'm waiting anyway as the user ID isn't in the right groups
   "Unknown owner on line 33: make sure @vaavva exists and has write access to the repository"